logos/src/logos/main.py

```
@@ -0,0 +1,286 @@
+    with DBManager() as db:
+        return db.import_from_json(**data.dict())
+
+
+@app.get("/forward_host")
+def route_handler(request: Request):
+    host = request.headers.get("x-forwarded-host") or request.headers.get("forwarded")
+    return {"host": host}
+
+
+@app.post("/logosdb/add_billing")
+async def import_json(data: AddBillingRequest):
```

Ruff (0.11.9)

Redefinition of unused `import_json` from line 195

(F811)